### ISSUE TITLE
Object slips from end-effector

### ISSUE THREAD

**Issue by chisarie** [deepmind/mujoco#239]
Hi, 

I have a franka emika panda model and I wrote a short script to make it pick up a small cube. The problem is that this cube slips away from the end-effector, see the attached video. I tried playing around with some friction parameters, but I did not manage to get it work. Do you have any suggestions?

The cube is modeled as 
```xml
<body name="cube" pos="1 1 1">
    <joint name="cube_joint" type="free" damping="1"></joint>
    <geom type="box" size=".02 .02 .02" mass="0.1" rgba="0 0.7 0 1.0" />
</body>
```

Thank you very much in advance!

https://user-images.githubusercontent.com/43339851/163166523-a234ba28-67c5-4ccb-bab1-ff1f6df9bdaa.mp4

**Comment by yuvaltassa** [comment]
What is the colliding geom on the tip of your end-effector?

If it is the actual mesh I would strongly recommend to disable the collisions for the mesh and add a layer of (possibly invisible) box primitives that overlap with the end effector.

**Comment by chisarie** [comment]
Hi Yuval,

thank you very much for the tip. Yes, the colliding geom was the actual mesh. I now changed it to box primitives as suggested and the result is already much better. 
Nevertheless, there is still a slow slipping happening, which means that the block is still falling from the gripper, just more slowly.
I tried playing with friction on both the object and the primitive gripper boxes by setting for example `friction="2 0.005 0.0001"`, but this resulted in very bad penetration, i.e. the gripper just went through the object.

Is there any other way I can make the object or the gripper box primitive "stickier"?

Thank you again!
